src/main/java/thePackmaster/patches/intriguepack/ExultationPatch.java

```diff
@@ -0,0 +1,58 @@
+package thePackmaster.patches.intriguepack;
+
+import basemod.ReflectionHacks;
+import com.evacipated.cardcrawl.modthespire.lib.SpirePatch;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.vfx.FastCardObtainEffect;
+import com.megacrit.cardcrawl.vfx.cardManip.ShowCardAndObtainEffect;
+import thePackmaster.cards.intriguepack.Exultation;
+
+public class ExultationPatch {
+
+    @SpirePatch(
+            clz = FastCardObtainEffect.class,
+            method = "update"
+    )
+    public static class OnPickupCardDoStuffPatch2 {
+        public OnPickupCardDoStuffPatch2() {
```

Comment:
Very minor, but you can delete this constructor -- it's not doing anything so it's not needed.

Same for the constructor of the other patch below.